<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PcdDialogForm</class>
 <widget class="QDialog" name="PcdDialogForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>344</width>
    <height>238</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ponto</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_7">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Arquivo:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2" colspan="4">
       <widget class="QLineEdit" name="ledColPointFileName"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Latitude:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2" colspan="4">
       <widget class="QLineEdit" name="ledColPointLatitude"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Longitude:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2" colspan="4">
       <widget class="QLineEdit" name="ledColPointLongitude"/>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QLabel" name="label_56">
        <property name="text">
         <string>Fuso Horário:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2" colspan="4">
       <widget class="QComboBox" name="timeZoneCmb">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <item>
         <property name="text">
          <string>+12:00</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>+11:00</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>+10:00</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>+09:00</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>+08:00</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>+07:00</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>+06:00</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>+05:00</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>+04:00</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>+03:00</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>+02:00</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>+01:00</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>+00:00</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>-01:00</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>-02:00</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>-03:00</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>-04:00</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>-05:00</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>-06:00</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>-07:00</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>-08:00</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>-09:00</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>-10:00</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>-11:00</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>-12:00</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QCheckBox" name="activeCheckBox">
        <property name="text">
         <string>Ativo</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Projeção:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="5">
       <widget class="QPushButton" name="projectionBtn">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Alterar a projeção dos dados&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset theme="projection">
          <normaloff>.</normaloff>.</iconset>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>65</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="5" column="1" colspan="3">
       <widget class="QPushButton" name="okBtn">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Salvar alterações&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>OK</string>
        </property>
        <property name="icon">
         <iconset theme="confirm">
          <normaloff>.</normaloff>.</iconset>
        </property>
       </widget>
      </item>
      <item row="5" column="4" colspan="2">
       <widget class="QPushButton" name="cancelBtn">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8pt;&quot;&gt;Cancelar alterações&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>Cancelar</string>
        </property>
        <property name="icon">
         <iconset theme="cancel">
          <normaloff>.</normaloff>.</iconset>
        </property>
       </widget>
      </item>
      <item row="4" column="3" colspan="2">
       <widget class="QLineEdit" name="projectionTxt">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
